config/config.go

```diff
@@ -902,18 +902,22 @@ func escapeEnv(value string) string {
 }
 
 func loadConfig(config string) ([]byte, error) {
-	u, err := url.Parse(config)
+	content, err := ioutil.ReadFile(config)
 	if err != nil {
-		return nil, err
-	}
+		u, err := url.Parse(config)
+		if err != nil {
+			return nil, err
+		}
 
-	switch u.Scheme {
-	case "https", "http":
-		return fetchConfig(u)
-	default:
-		// If it isn't a https scheme, try it as a file.
+		switch u.Scheme {
+		case "https", "http":
+			return fetchConfig(u)
+		default:
+			// If it isn't a https scheme, an error occurred
```

Comment:
In this case, the actual return of `loadConfig` would be `nil, nil`, which is not a good situation, please provide a meaningful error message or reuse the error from `ioutil.ReadFile`. (the `err` from that one is overridden by the `err` from `url.Parse`)